scripts.py

```diff
@@ -204,11 +212,15 @@ def __init__(self, *args, **kwargs):
         super(TimestampScript, self).__init__(*args, **kwargs)
         self.timestamp_collection = None
 
-    def update_timestamp(self):
+    def update_timestamp(self, start_time, exception):
         """Update the appropriate Timestamp for this script."""
-        Timestamp.stamp(self._db, self.script_name,
-                        collection=self.timestamp_collection)
-
+        Timestamp.stamp(
+            _db=self._db, service=self.script_name,
+            service_type=Timestamp.SCRIPT_TYPE,
+            collection=self.timestamp_collection,
+            start=start_time, exception=exception
+        )
+        
```

Comment:
Changing this method ensures that all Timestamps created by scripts have the right `service_type`.